sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventProcessorClientTest.java

```diff
@@ -200,9 +202,10 @@ public void testProcessSpans() throws Exception {
             .createConsumer(anyString(), anyInt()))
             .thenReturn(consumer1);
         when(eventData1.getSequenceNumber()).thenReturn(1L);
-        when(eventData2.getSequenceNumber()).thenReturn(2L);
+        when(eventData1.getSequenceNumber()).thenReturn(2L);
```

Comment:
Did you mean to change it from eventData2 -> eventData1? The same event data returns two different sequence values?

Reply:
Was meaning to remove eventData2 altogether.  Removed now.